omod/src/main/java/org/openmrs/module/referenceapplication/filter/RequireLoginLocationFilter.java

```diff
@@ -94,18 +94,14 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 	}
 	
 	/**
-	 * Determines if the filter should be skipped for the specified request uri, typically all requests
-	 * for static content, non ref app pages, login and logout are be ignored.
+	 * Determines if the filter should be skipped for the specified request uri.
+	 * in config.xml file, the filter-mapping is configured to match *.page  and /
 	 * 
 	 * @param requestUri the request uri to check
 	 * @return true if the filter should be skipped otherwise false
 	 */
 	private boolean skipFilter(String requestUri) {
-		if (requestUri.endsWith(".page")) {
-			return ArrayUtils.indexOf(allowedRequestURIs, requestUri) > -1;
-		}
-		
-		return true;
+		return ArrayUtils.indexOf(allowedRequestURIs, requestUri) > -1;
```

Comment:
Did you verify that the login and logout features still work?

Reply:
Yes I did. I also checked some scenarii with 2 tabs :
- login in one tab, try to reach the home page from another one
- logout from on tab, refresh the other tab to check that the user is logged out